Android.mk

```diff
@@ -370,8 +388,43 @@ LOCAL_SRC_FILES                            := \
     src/ncp/ncp_base_dispatcher.cpp           \
     src/ncp/ncp_buffer.cpp                    \
     src/ncp/ncp_uart.cpp                      \
+    $(NULL)
+
+include $(BUILD_STATIC_LIBRARY)
+
+include $(CLEAR_VARS)
+
+LOCAL_MODULE := ot-ncp
```

Comment:
```suggestion
ifneq ($(USE_OTBR_DAEMON), 1)
LOCAL_MODULE := ot-ncp
```

Reply:
We will always build ot-ncp. The change here is to have ot-ncp depend on libopenthread-ncp rather than compile all the files under `src/ncp`.